components/brave_wallet/browser/pref_names.h

```diff
@@ -104,4 +104,7 @@ inline constexpr char kBraveWalletUserAssetsDeprecated[] =
 inline constexpr char kBraveWalletTransactionsDBFormatMigrated[] =
     "brave.wallet.transactions_db_format_migrated";
 
+inline constexpr char kBraveWalletPrivateWindowsEnabled[] =
+    "brave.wallet.private_windows_enabled";
+
```

Comment:
Please put this next to `kAutoPinEnabled`. This bottom section is for deprecated prefs